### PR TITLE
Global Footer

### PR DESCRIPTION
**Pull request by alexkilgour**
## Introduce the new global footer component

This replicates the functionality from `global-corporate-footer` and adds a product-specific links section above the corporate section. This is intended to replace the corporate footer component and any brand-specific footer components.

A future pull request will archive the older footer components and link to this one.

### CODE REVIEW

**Review by amyhupe — DISMISSED** [review]
Looks good - just one comment from me - looks like there's a rogue heading...

File: toolkits/global/packages/global-footer/README.md [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+
+## Template
+
+Find a configurable template in the [`view` folder](https://github.com/springernature/frontend-toolkits/tree/master/toolkits/global/packages/global-footer/view).
+
+You can see an example in the [`demo` folder](https://github.com/springernature/frontend-toolkits/tree/master/toolkits/global/packages/global-footer/demo).
+
+### Template configuration
+
+#### Using the Corporate footer
```
[the remark this answers is not in the document]
Comment (alexkilgour) [reply toolkits/global/packages/global-footer/README.md:144]: good spot, thanks 👍

**Comment by benjclark** [comment]
There is varying indentation used in the changed files. They should all use the same indentation?

**Comment by sturobson** [comment]
If this replaces the `global-corporate-footer` as suggested in the changelog - should there be a separate PR to deprecate that component?

**Comment by alexkilgour** [comment]
> If this replaces the `global-corporate-footer` as suggested in the changelog - should there be a separate PR to deprecate that component?

Yep, there will be 👍

File: toolkits/global/packages/global-footer/scss/50-components/footer.scss [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+	}
+
+	@supports (display: grid) {
+		padding-right: 0;
+		max-width: none;
+		margin-bottom: 0;
+	}
+}
+
+.c-footer__group--separator {
```
Comment (sturobson) [inline toolkits/global/packages/global-footer/scss/50-components/footer.scss:86]: 💡
For the "product only footer"(?) bits do we want to expose a fresh variable for Sass to determine wether to compile this code or not?

```
@if variable-exists($product-footer--exists) {
    /* The CSS for the product footer */
}
```

Then folks would need to include `$product-footer--exists: true` in their config.

Comment (sturobson) [reply toolkits/global/packages/global-footer/scss/50-components/footer.scss:86]: every byte counts :D 

but I think this is a larger architectural and 'how we write CSS' question that could be had which shouldn't block this PR :)

**Review by sturobson — DISMISSED** [review]
LGTM!

**Pull request merged**